qa/L0_perf_nomodel/run_test.sh

```diff
@@ -42,26 +41,38 @@ TENSOR_SIZE=${TENSOR_SIZE:=1}
 SHARED_MEMORY=${SHARED_MEMORY:="none"}
 REPORTER=../common/reporter.py
 
-DATADIR=/data/inferenceserver/${REPO_VERSION}
+DATADIR=${DATADIR:="/data/inferenceserver/${REPO_VERSION}"}
 RESULTDIR=${RESULTDIR:=.}
 
-SERVER=/opt/tritonserver/bin/tritonserver
-SERVER_ARGS="--model-repository=`pwd`/models"
+OPTDIR=${OPTDIR:="/opt"}
+SERVER=${OPTDIR}/tritonserver/bin/tritonserver
+BACKEND_DIR=${OPTDIR}/tritonserver/backends
+SERVER_ARGS="--model-repository=`pwd`/models --backend-directory=${BACKEND_DIR}" 
 source ../common/util.sh
 
+if [ "$TEST_JETSON" -eq 1 ]; then
+    ARCH="aarch64"
+    PERF_CLIENT=${OPTDIR}/tritonserver/clients/bin/perf_client
+else
+    ARCH="x86_64"
+    PERF_CLIENT=../clients/perf_client
+fi
+
 # Select the single GPU that will be available to the inference server
 export CUDA_VISIBLE_DEVICES=0
 
 mkdir -p ${RESULTDIR}
 RET=0
 
-cp /opt/tritonserver/backends/python/triton_python_backend_utils.py .
+if [ "$TEST_JETSON" -eq 0 ]; then
```

Comment:
Check if the backends are under test and prepare the model accordingly instead of checking TEST_JETSON.

Reply:
Done